plugins/fireeye_hx/plugin.spec.yaml

```diff
@@ -305,7 +306,85 @@ types:
     subtype:
       title: Subtype
       description: Alert subtype
-      type: object
+      type: string
+      required: false
+
+  user:
+    _id:
+      title: ID
+      description: ID
+      type: integer
+      required: false
+    username:
+      title: Username
+      description: Username
+      type: string
+      required: false
+  data:
+    _id:
+      title: ID
+      description: ID
+      type: string
+      required: false
+    last_sysinfo:
+      title: Last System Info
+      description: Last system info
+      type: string
+      required: false
+    requested_by_actor:
+      title: Requested By Actor
+      description: Requested by actor
+      type: user
+      required: false
+    requested_on:
+      title: Requested On
+      description: Requested on
+      type: string
+      required: false
+    contained_by_actor:
+      title: Contained By Actor
+      description: Contained by actor
+      type: user
+      required: false
+    contained_on:
+      title: Contained On
+      description: Contained on
+      type: string
```

Comment:
Shouldn't this be a date?

Reply:
`contained_on` is part of the custom output and we shouldn't change its type.